_config/graphql.yml

```diff
@@ -16,3 +16,12 @@ SilverStripe\GraphQL\Manager:
             fields: [ID, FirstName, Surname]
             operations:
               readOne: true
+    default:
```

Comment:
Hmmm, this is introducing a GraphQL API in to every project using the CMS now. What are the implications of that? Is there auth that only applies to the `admin` API?

Reply:
No.. the `default` schema was always pre-configured for you, but it doesn't actually activate until you route it. All this configuration doesn't turn into anything executable until this happens:

```
SilverStripe\Control\Director:
  rules:
    'graphql': '%$SilverStripe\GraphQL\Controller.default'
```